ReactiveUI/AddTo.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace System.Reactive.Disposables
+{
+    public static class CompositeDisposableExtensions
+    {
+        /// <summary>
+        /// Adds a disposable to the specified <see cref="CompositeDisposable"/>.
+        /// </summary>
+        /// <typeparam name="T">
+        /// The type of the disposable.
+        /// </typeparam>
+        /// <param name="this">
+        /// The disposable.
+        /// </param>
+        /// <param name="compositeDisposable">
+        /// The <see cref="CompositeDisposable"/> to which <paramref name="this"/> will be added.
+        /// </param>
+        /// <returns>
+        /// The disposable.
+        /// </returns>
+        public static T AddTo<T>(this T @this, CompositeDisposable compositeDisposable)
```

Comment:
@gluck I think there's a mistake here, the function returns the original value, not the disposable, it's just the comment that is wrong